diff --git a/third-party-podspecs/fmt.podspec b/third-party-podspecs/fmt.podspec
index 121be5011de5e34ceb526f4ea60e085a595676d0..99cf21ab86c907e9ee8264cafb2b35f8f86c1c04 100644
--- a/third-party-podspecs/fmt.podspec
+++ b/third-party-podspecs/fmt.podspec
@@ -8,14 +8,14 @@ fmt_git_url = fmt_config[:git]
 
 Pod::Spec.new do |spec|
   spec.name = "fmt"
-  spec.version = "11.0.2"
+  spec.version = "11.1.4"
   spec.license = { :type => "MIT" }
   spec.homepage = "https://github.com/fmtlib/fmt"
   spec.summary = "{fmt} is an open-source formatting library for C++. It can be used as a safe and fast alternative to (s)printf and iostreams."
   spec.authors = "The fmt contributors"
   spec.source = {
     :git => fmt_git_url,
-    :tag => "11.0.2"
+    :tag => "11.1.4"
   }
   spec.pod_target_xcconfig = {
     "CLANG_CXX_LANGUAGE_STANDARD" => rct_cxx_language_standard(),
